components/ui/Paragraph.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+
+const Paragraph = styled.p`
+    color: #000000;
```

Comment:
Please extract this color as CSS custom properties to `GlobalStyles` and use `var(--text-color)` here instead.